sci-libs/symengine/symengine-0.1.0.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=5
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit cmake-utils python-single-r1
+
+DESCRIPTION="Fast symbolic manipulation library, written in C++"
+HOMEPAGE="https://github.com/sympy/symengine"
+SRC_URI="https://github.com/sympy/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+IUSE="boost openmp python threads"
+REQUIRED_USE="python? ( ${PYTHON_REQUIRED_USE} )"
+
+DEPEND="
+	dev-libs/jemalloc
+	boost? ( dev-libs/boost )
+	python? (
+		dev-python/cython[${PYTHON_USEDEP}]
+		dev-python/setuptools[${PYTHON_USEDEP}] )"
+RDEPEND="${DEPEND}
```

Comment:
Good that you added the missing dep, but now you are depending during runtime on the buildtime deps. I would recommend to switch it around and have ```DEPEND="${RDEPEND}```